packages/vue/src/components/SpAssets/SpAssets.vue

```diff
@@ -57,26 +76,74 @@ export default defineComponent({
       )
     })
 
+    const filteredBalanceList = computed(() => {
+
+      let searchArray = balances.value,
+        searchString = state.searchQuery
+
+      console.log({searchString})
+
+      if (!searchString) {
+        return sortList(searchArray)
+      }
+
+      searchString = searchString.trim().toLowerCase()
+
+      searchArray = searchArray.filter((item) => {
+        if (item.denom.toLowerCase().indexOf(searchString) !== -1) {
```

Comment:
maybe it's better to use `.includes` so to not depend on whole words?

Reply:
Working on refactoring this functionality.

Reply:
shall we use includes?

Reply:
correction: `indexOf` works just fine, i.e. it is as capable as `includes`

Reply:
I will use `includes` as it looks more clear.